ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -773,6 +773,17 @@ literals:
 - mixed_air_relative_humidity_sensor
 - mixed_air_dewpoint_temperature_sensor
 
+#Heat Wheel Telemetry Fields
+- heat_wheel_speed_percentage_sensor
```

Comment:
Should we create a heat wheel abstract type to support the AHU defined?

Reply:
@tasodorff That's probably a good idea. Heat wheel AHUs are pretty common and there will probably be more in the future. 
What should the abstract typ e be called?

Reply:
HTWHLSPC might be a good one for heat wheel speed percentage command, run command, and run status.